client/src/components/QA/Questions/QuestionList.jsx

```diff
@@ -3,10 +3,21 @@ import QListCSS from './QuestionList.module.css';
 import QuestionItem from './QuestionItem.jsx';
 
 function QuestionList(props) {
+  const { questions } = props;
   return (
-    <div className={QListCSS.questionlist}>
-      {props.questions.map((question, index) => <QuestionItem question={question} key={index} />)}
+    <div>
+      <div className={QListCSS.questionlist}>
+        {questions.map((question) => (
+          <QuestionItem
+            question={question}
+            key={question.question_id}
+          />
+        ))}
+      </div>
+      <button type="submit" className={QListCSS.moreButton}>More Answered Questions</button>
```

Comment:
@amberlyM Why not put these two in a single div and align them to the left of the page like in the example visual?